google/cloud/sql/connector/InstanceConnectionManager.py

```diff
@@ -78,12 +103,44 @@ def __init__(
                 "Arg instance_connection_string must be in "
                 + "format: project:region:instance."
             )
+
         self._loop = loop
         self._auth_init()
         self._priv_key, self._pub_key = generate_keys()
+        self._pub_key = self._pub_key.decode("UTF-8")
+        self._mutex = threading.Lock()
+
+        async def create_client_session():
+            return aiohttp.ClientSession()
+
+        self._client_session = asyncio.run_coroutine_threadsafe(
+            create_client_session(), loop=self._loop
+        ).result()
+
+        logging.debug("Updating instance data")
```

Comment:
Make sure to change these 
```suggestion
        logger.debug("Updating instance data")
```